Revert "Update question.md" (#7183)
hashicorp/consul: .github/ISSUE_TEMPLATE/question.md

---
name: Question

> about: If you have a question, please check out our other community resources instead of opening an issue.

Reviewer: What if we were a bit more explicit:
```suggestion
about: If you have a question, please do not create an issue. Instead check out our other community resources.
```
